components/Table/TablePercent.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import ReactTooltip from 'react-tooltip';
+import styles from './TablePercent.css';
+
```

Comment:
If we declare the function inside the stateless component then each re-render will trigger the creation of a new function. Not really the worst thing in the world but it could become memory intensive in some situations with very large tables (assuming those tables aren't paginated in some way).